vertexing/firmware/bram_hist_vtx.cpp

```diff
@@ -0,0 +1,43 @@
+#include "bram_hist_vtx.h"
+
+void bhv_add_track(zbin_vt zbin, pt_t tkpt, ptsum_t hist[BHV_NBINS]) {
+    #pragma HLS pipeline II=2
+    #pragma HLS interface ap_memory port=hist
```

Comment:
For my own education, you would take the 2 IPs (bhv_add_track and bhv_find_pv) and in VHDL wire them to the same BRAM?

Reply:
yes, they're meant to use the two ports of one bram.
i'll make a few slides for our meeting nrxt tuesday